.editorconfig

```diff
@@ -0,0 +1,19 @@
+# top-most EditorConfig file
```

Comment:
Why is this being added? Also, is this something that can be used across different IDEs?